packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -211,58 +266,87 @@ class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget> {
     Size renderSize,
     Offset renderOffset,
   }) {
+    Widget buildEntranceAnimation(BuildContext context, Widget child) {
+      final vertical = properties.isMainAxisVertical;
+      final horizontal = properties.isMainAxisHorizontal;
+      Size size = renderSize;
+      if (childProperties.flexFactor != null) {
+        size = SizeTween(
+          begin: Size(
+            horizontal ? minRenderWidth - entranceMargin : renderSize.width,
+            vertical ? minRenderHeight - entranceMargin : renderSize.height,
+          ),
+          end: renderSize,
+        ).evaluate(expandedEntrance);
+      }
+      return Opacity(
+        opacity: min([allEntrance.value * 5, 1.0]),
+        child: Padding(
+          padding: EdgeInsets.symmetric(
+            horizontal: (renderSize.width - size.width) / 2,
+            vertical: (renderSize.height - size.height) / 2,
```

Comment:
very nit: * 0.5 instead of / 2 for consistency ?

Reply:
There's nothing wrong with using division to express division.

I'm also not sure where else there's a *0.5 in this PR.

Reply:
it's in `inspector_data_models.dart` where I do the computation for the empty space. But yes it's fine